recipes-kernel/linux/linux-nilrt-current.inc

```diff
@@ -0,0 +1,6 @@
+NI_RELEASE_VERSION = "master"
+LINUX_VERSION = "4.14"
+KBRANCH = "nilrt/${NI_RELEASE_VERSION}/${LINUX_VERSION}"
+GIT_KERNEL_REPO = "linux.git"
+
+require linux-nilrt.inc
```

Comment:
Should there be a newline at the end?

Reply:
I don't think it's needed. Do you know of places this would cause an issue?

Reply:
I don't think there'll be any issues - just mentioned as its a convention on linux machines to end files with newline. I don't mind you dropping this though.

Reply:
OK, fair point. I pushed a fixup commit just for consistency.